src/styles.scss

```diff
@@ -1506,3 +1511,13 @@ hr {
 .mat-radio-button .mat-form-field-can-float.mat-form-field-should-float .mat-form-field-label {
   top: 1.8em;
 }
+
+// Style home search mat-form-field
+// Make mat-form-field match home search bar, could look into more solutions
+.home-field .mat-form-field-wrapper {
+  padding-bottom: 0;
+}
+.home-field .mat-form-field-infix {
+  padding: 0.8em 0 0.8em 0;
+  border-top: 0em;
+}
```

Comment:
umm....if it's a style that's specific to a component (because it can't be generalized), then it should be in the component scss. In this case, there's some weird problems going on and it's unlikely to be occur in a future component so we won't ever use the class elsewhere.